lambdatimes/src/components/Content/Tab.js

```diff
@@ -1,22 +1,44 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-const Tab = props => {
+ const Tab = props => {
+  if (this.props.tab === selectedTab) {
+    return className='tab active-tab';
+  } else {
+    return className='tab';
+  }
   /* Using your props, determine if the `tab` prop matches the `selectedTab` prop, 
       if they match, the className should be: 'tab active-tab', 
       if it is not it should just be 'tab'*/
   return (
     <div
-      className={''}
+      className={'tab'}
```

Comment:
You will want to conditionally set the className here to either be "tab active-tab" or "tab" based on whether the tab prop matches the selectedTab prop.